web/apps/labelstudio/src/pages/CreateProject/Config/Preview.jsx

```diff
@@ -31,18 +34,18 @@ export const Preview = ({ config, data, error, loading, project }) => {
    * @param {*} _ LS instance
    * @param {string} url http/https are not proxied and returned as is
    */
-  const onPresignUrlForProject = async (_, url) => {
+  const onPresignUrlForProject = useCallback(async (_, url) => {
```

Comment:
if there are no dependencies - do we still need to use `useCallback`?

Reply:
Technically in this case not really ... it would be more a problem if it was being used as a chained dependency into a useEffect, which I'm avoiding by making the only value it has for implicit state `project` use a ref.